src/DotNetWorker.Core/DotNetWorker.Core.csproj

```diff
@@ -2,15 +2,14 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks>net5.0;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net5.0;net8.0;net9.0;netstandard2.0</TargetFrameworks>
```

Comment:
I did not remove net5.0 as part of this PR. We have a GH work item to track that change https://github.com/Azure/azure-functions-dotnet-worker/issues/1095